### ISSUE TITLE
Right nested menu not scrolling when resolution set low

### ISSUE THREAD

**Issue by videosmith** [Cuperino/QPrompt-Teleprompter#111]
**Bug Description**
When using a lower resolution, like 480p and in Full Screen mode, the right nested menu will not scroll like the left nested menu

**Steps to Reproduce**
1.  Set OS to 480p
2.  Launch QPrompt 1.1.0
3.  Enter Full Screen mode
4.  Select nested right menu  (a graphic identifying the hotspot might be helpful, see pic)
5.  Menu is frozen at current position.

**Expected behavior**
Should scroll like the left nested menu

**Detailed description of display configuration** (if applicable)
480p HDMI 0 Raspberry Pi400

**Screenshots** (optional)
![Screenshot from 2022-08-06 15-32-52](https://user-images.githubusercontent.com/88691671/183270700-54d37aa5-52a6-4797-8713-09e0fcf450de.png)
![Screenshot from 2022-08-06 15-35-52](https://user-images.githubusercontent.com/88691671/183270730-8105a0ec-2dac-4381-a087-fc60c27b8642.png)
![Screenshot from 2022-08-06 15-38-47](https://user-images.githubusercontent.com/88691671/183270732-8ac333f4-b160-4308-81d2-93dd10c0d476.png)

**Device information**
 - QPrompt Version: qprompt-v1.1-8fc947c-linux-aarch64.deb
 - Operating System: Raspberry Pi Bullseye 64, latest updates
 - libqhotkey-1.5.0-linux-aarch64.deb

**Additional context**

**Comment by Cuperino** [comment]
Got it. I may not be able to fix this, as the menu component in itself is a part of the Kirigami framework and not managed by QPrompt.

**Comment by videosmith** [comment]
Can the nested right-menu items be merged with the nested left-menu since that one scrolls?

**Comment by Cuperino** [comment]
Not really... I mean, it can be done but that would be a very hacky workaround. I'm fairly confident that this change can be done upstream, it just takes a bit of time... The KDE community is very open to contributions, and its developers are about as accessible as me when the right channels are used.

**Comment by Cuperino** [comment]
And this is an improvement that would benefit the users of more software, not just QPrompt.

**Comment by videosmith** [comment]
Understood.  Since the left nested menu scrolls, can the right menu be merged with that one?
    On Sunday, August 7, 2022, 01:19:21 PM EDT, Javier O. Cordero Pérez ***@***.***> wrote:  
 
 


Got it. I may not be able to fix this, as the menu component in itself is a part of the Kirigami framework and not managed by QPrompt.

**Comment by Cuperino** [comment]
> Since the left nested menu scrolls, can the right menu be merged with that 

It could, but they shouldn't be merged.

The right menu is a context menu, that changes depending on which part of the program you're at; the left is a global menu available throughout the entire program. Since the items in the right menu are context dependent, if a context were to replace the prompter's context at runtime, it would cause the resources needed by the left menu to become inaccessible, leading to potential crashes. One could always check whether a resource is available and disable or hide those inputs if not, but that could cause confusion to some users and makes the program more difficult to maintain.

**Comment by Cuperino** [comment]
The Qt 6 version of Kirigami removed the ability to show this menu, so this issue can't be fixed because it doesn't exist anymore.
